cinch/roles/jenkins_common/defaults/main.yml

```diff
@@ -5,8 +5,11 @@ jenkins_ssh_keyfile: id_rsa
 # supported beyond the EOL support for their packages in the distribution of
 # choice
 java_version: 8
-# Needed for certain OpenShift instances
```

Comment:
Can you add this default value into the inventory that is used in generating the Docker containers?